client/components/EyesWanted/AddEyesWanted.vue

```diff
@@ -39,20 +39,26 @@
     },
     data() {
       return {
-        // thanks: '',
+        eyeswantedId: '',
         alerts: {} // Displays success/error messages encountered during freet modification
       };
     },
     methods: {
       existingEyesWanted() {
         /**
-         * Return if user has thanked the update
+         * Return if user has added eye wanted the update
          */
         const alleyeswanted = this.$store.state.alleyeswanted;
-        console.log(alleyeswanted[1].updateId);
         const exists = alleyeswanted
-                        .filter(alleyeswanted => alleyeswanted.upateId === this.update._id)
+                        .filter(alleyeswanted => alleyeswanted.update._id === this.update._id)
                         .length === 1;
+        if (exists) {
+          this.eyeswantedId = alleyeswanted
+                        .filter(alleyeswanted => alleyeswanted.update._id === this.update._id)[0]._id;
+        };
+        console.log(exists);
+        console.log(alleyeswanted
+                        .filter(alleyeswanted => alleyeswanted.update._id === this.update._id))
         return exists;
```

Comment:
make this a computed(){} property, same goes for `existingThanks()` as well. this should hopefully make it so that you don't need to refresh it every time